src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -262,6 +279,22 @@ export default function TransactionFeedV2() {
     [isFetching, data?.transactions, originalArgs?.endCursor, standByTransactions.confirmed]
   )
 
+  /**
+   * In order to avoid bloating stand by transactions with confirmed transactions that are already
+   * present in the feed via pagination – we need to cleanup them up. This must run for every page
+   * as standByTransaction selector might include very old transactions. We should use the chance
+   * whenever the user managed to scroll to those old transactions and remove them from persisted
+   * storage. Maybe there is a better way to keep it clean with another saga watcher?
+   */
+  useEffect(
+    function cleanupStandByTransactions() {
+      if (data?.transactions.length) {
+        dispatch(removeDuplicatedStandByTransactions(data.transactions))
+      }
```

Comment:
Instead of using an effect and a new action, can we react on the RTK-Query API fetched action?
Directly at the reducer level.

Reply:
@jeanregisser Will investigate and get back to you on this!